bin/check-rabbitmq-messages.rb

```diff
@@ -39,9 +40,9 @@ class CheckRabbitMQMessages < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i),
          default: 15_672
 
-  option :user,
+  option :username,
```

Comment:
I like making it consistent but this makes it a breaking change. This will need to be called out in the CHANGELOG.